ckanext/dcatapchharvest/tests/test_swiss_schemaorg_profile_serialize.py

```diff
@@ -164,8 +163,8 @@ def test_graph_from_dataset(self):
         assert self._triple(g, dataset_ref, SCHEMA.identifier, extras['identifier'])
 
         # Dates
-        assert self._triple(g, dataset_ref, SCHEMA.datePublished, dataset['metadata_created'])
-        assert self._triple(g, dataset_ref, SCHEMA.dateModified, dataset['metadata_modified'])
+        assert self._triple(g, dataset_ref, SCHEMA.datePublished, dataset['issued'])
+        assert not self._triple(g, dataset_ref, SCHEMA.dateModified, None)
```

Comment:
i feel like there ought to be a better way to test this, because we want to assert there's no triple at all with `subject=dataset_ref` and `predicate=SCHEMA.dateModified`, whether the `object` is `None` or anything else... could we do something like
```
assert len(g.objects(dataset_ref, SCHEMA.dateModified)) == 0
```
?